noir-projects/aztec-nr/aztec/src/context/avm_context.nr

```diff
@@ -82,7 +82,7 @@ impl PublicContextInterface for AvmContext {
         nullifier_exists(unsiloed_nullifier, address.to_field()) == 1
     }
 
-    fn emit_unencrypted_log<T>(&mut self, log: T) {
+    fn emit_unencrypted_log<T,N,M>(&mut self, log: T) {
```

Comment:
The AVM context and interface was only changed to allow the public and private contexts to use traits in `logs_traits` (these define fixed lengths for many types, so we can sha hash inside the circuit). These don't need to be carried over!